prime-router/settings/staging/0158-al-phl-and-baptist-onboarding.yml

```
@@ -0,0 +1,215 @@
+        reportingFacilityName: null
+        reportingFacilityId: null
+        reportingFacilityIdType: null
+        suppressQstForAoe: false
+        suppressHl7Fields: null
+        suppressAoe: false
+        defaultAoeToUnknown: false
+        replaceUnicodeWithAscii: false
+        useBlankInsteadOfUnknown: null
+        truncateHDNamespaceIds: false
+        truncateHl7Fields: null
+        usePid14ForPatientEmail: false
+        convertTimestampToDateTime: null
+        cliaForOutOfStateTesting: null
+        cliaForSender: {}
+        phoneNumberFormatting: "STANDARD"
+        suppressNonNPI: false
+        processingModeCode: null
+        replaceDiiWithOid: null
+        applyOTCDefault: false
+        useOrderingFacilityName: "STANDARD"
+        valueSetOverrides: {}
+        nameFormat: "standard"
+        receivingOrganization: null
+        convertPositiveDateTimeOffsetToNegative: false
+        stripInvalidCharsRegex: null
+        convertDateTimesToReceiverLocalTime: false
+        useHighPrecisionHeaderDateTimeFormat: false
+        type: "HL7"
+        truncationConfig:
+          truncateHDNamespaceIds: false
+          truncateHl7Fields: []
+          customLengthHl7Fields: {}
+      jurisdictionalFilter:
+        - "(Bundle.entry.resource.ofType(MessageHeader).event.code = 'O21')"
+        - "(%performerState.exists() and %performerState = 'AL') or (%patientState.exists() and %patientState = 'AL')"
+      qualityFilter:
+        - "true"
+      routingFilter: []
+      processingModeFilter: []
+      reverseTheQualityFilter: false
+      conditionFilter: []
+      deidentify: false
+      deidentifiedValue: ""
+      timing:
+        operation: "MERGE"
+        numberPerDay: 1440
+        initialTime: "00:00"
+        timeZone: "EASTERN"
+        maxReportCount: 100
+        whenEmpty:
+          action: "NONE"
+          onlyOncePerDay: false
+      description: ""
+      transport: !<SFTP>
+        host: "172.17.6.20"
+        port: "22"
+        filePath: "./upload"
+        credentialName: null
+        type: "SFTP"
+      externalName: "Alabama State Public Health Lab ETOR Receiver"
+      timeZone: null
+      dateTimeFormat: "OFFSET"
+#   BAPTIST Sender (etor-nbs-orders) sends to ETOR-TI Receiver (etor-service-receiver)
+#   ETOR-TI Sender (etor-service-sender) sends to AL-PHL Receiver (etor-nbs-results)
+- name: "baptist"
+  description: "Baptist Alabama Public Health Lab"
+  jurisdiction: "STATE"
+  stateCode: "AL"
+  countyName: null
+  filters: []
+  featureFlags: []
+  keys: null
+  senders:
+    - !<full-elr>
+      name: "etor-nbs-results"
+      organizationName: "baptist"
+      format: "HL7"
+      customerStatus: "active"
+      schemaName: ""
+      processingType: "sync"
+      allowDuplicates: true
+      senderType: null
+      primarySubmissionMethod: null
+      topic: "etor-ti"
+      version: null
+      createdBy: null
+      createdAt: null
+  receivers:
+    - name: "etor-nbs-orders"
```

Baptist is an HCO, and they receive results.

```suggestion
    - name: "etor-nbs-results"
```